app/public/worker.js

```diff
@@ -64,6 +67,10 @@ self.onmessage = function (e) {
     ]);
   }
 
+  for (let data_ptr of font_datas) {
+    Module._free(data_ptr);
```

Comment:
This is a nit, but technically this is is freeing the data before `FT_Done_Face` is called in `free_fontstack`, so maybe a bit more correct to move this down? I don't know of any problem it would cause though.